examples/compare_compression.rs

```diff
@@ -64,6 +64,10 @@ fn main() {
                 .takes_value(true)
                 .possible_values(&["identity", "gzip", "br", "deflate"])
                 .help("Compression format for the binjs files"),
+            Arg::with_name("nodememory")
+                .long("BINAST_NODE_MAX_OLD_SPACE_SIZE")
```

Comment:
That name is a bit long :)

Reply:
yeah. Let me 🤔 over it

Reply:
Also, shouldn't you require it to be a number?

Reply:
It doesnt matter, anyway we have to typecast it later right ?

Reply:
Yes, but casting it earlier makes for better error messages/documentation.

Reply:
It's not very nice, but you can add something along the lines of

```rust
    .validator(|x| match x.parse<u32>() {
        Ok(_) => Ok(()),
        Err(e) => Err(format!("Invalid argument {:?}", e))
    })
```

to ensure that it's a proper integer.

Reply:
It is kinda repetitive, you have to write this thing everywhere, if we use validator 😞

Reply:
Maybe we could somehow make it part of `shift.rs`?

Reply:
Ok, you have convinced me: let's switch to plan B and make it an environment variable. This will be less invasive :)

Reply:
can we validate it inside shift ?

Reply:
ha ha, I didnt refresh. 

Environment variable 🎉 
But I have no idea about it. Let me explore

Reply:
1/ Perhaps Node already provides an Environment variable for that purpose? If so, we just need to document it.

2/ Otherwise, we can use an environment variable `BINAST_NODE_MAX_OLD_SPACE_SIZE` and use `std::env` to decode it in the `Shift` module.